src/rcheevos/consoleinfo.c

```diff
@@ -564,6 +564,25 @@ static const rc_memory_region_t _rc_memory_regions_msx[] = {
 };
 static const rc_memory_regions_t rc_memory_regions_msx = { _rc_memory_regions_msx, 1 };
 
+/* ===== MS DOS ===== */
+static const rc_memory_region_t _rc_memory_regions_ms_dos[] = {
+    /* DOS emulators split the 640 KB conventional memory into two regions.
+     * First the part of the conventional memory given to the running game at $000000.
+     * Then the remaining conventional memory which contains DOS and BIOS controlled
+     * memory at $100000. The length of this can vary depending on the hardware
+     * and DOS version (or emulated DOS shell).
+     * These first two regions will only ever total to 640 KB but the regions map
+     * to 1 MB each to make resulting memory addresses more readable.
```

Comment:
Would it be reasonable to put padding regions in?
```
    { 0x00000000U, 0x0009FFFFU, 0x00000000U, RC_MEMORY_TYPE_SYSTEM_RAM, "Game Conventional Memory" },
    { 0x000A0000U, 0x000FFFFFU, 0x00000000U, RC_MEMORY_TYPE_UNUSED, "Padding to align OS Conventional Memory" },
    { 0x00100000U, 0x0019FFFFU, 0x00100000U, RC_MEMORY_TYPE_SYSTEM_RAM, "OS Conventional Memory" },
    { 0x001A0000U, 0x001FFFFFU, 0x00100000U, RC_MEMORY_TYPE_UNUSED, "Padding to align Expanded Memory" },
```

Reply:
Yeah I think that makes sense. In the latest commit I added the padding regions and also reworked the comment above it a bit. Also clarified how a non-DOS game (PC Booter game) will be mapped.